analysis/exponential.lean

```diff
@@ -127,6 +127,22 @@ by simp [log, lt_irrefl]
 @[simp] lemma log_one : log 1 = 0 :=
 exp_injective $ by rw [exp_log zero_lt_one, exp_zero]
 
+noncomputable definition nth_root (x : ℝ) (n : ℕ) : ℝ := exp ((log x) / n)
+

```

Comment:
`nth_root` of `0` should be `zero` I think.

Reply:
even if n=0?

Reply:
Sure, the only reasonable solution to `x^0 = 0` would be `x = 0`, right? For all other `x` it is clear that `x^0 = 1`.

Reply:
Don't make me argue for 0^0 = 1 again...